src/dom/istext.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/dom/istext
+ */
+
+/**
+ * Checks if the object is a native DOM Text Node.
+ *
+ * @param {*} obj
+ * @returns {Boolean}
+ */
+export default function isText( obj ) {
+	return Object.prototype.toString.call( obj ) == '[object Text]';
```

Comment:
Not sure if it's better checking `nodeType` or calling `toString()`. IMO both options should be good here.

Reply:
Checking `nodeType` seems to be a too naive duck typing. Calling `toString()` is slightly safer. Eventually, we could switch to `instanceof` but starting with `toString()` seems to be simpler.

Reply:
Simple `instanceof` check easily breaks in iframes, so it would be more complicated to implement it correctly.

Reply:
BTW: I wonder why `isDomNode` instead of `isNode`? It's under `src/dom/`. If we go with `isText` we should go with `isNode` too.

Reply:
I forgot to comment on this – yep, `isNode()` would be better. @ma2ciek could you report a separate ticket for it and do the change?

Reply:
Sure, I'll handle it.

Reply:
https://github.com/ckeditor/ckeditor5-utils/issues/219